testing/xfpga/test_open_close_c.cpp

```diff
@@ -316,15 +316,16 @@ TEST_P(openclose_c_p, close_03) {
  *             already opened token returns FPGA_OK.
  */
 TEST_P(openclose_c_p, open_share) {
-  fpga_handle h1, h2;
+  fpga_handle h1 = NULL;
+  fpga_handle h2 = NULL;
 
   EXPECT_EQ(FPGA_OK, xfpga_fpgaOpen(tokens_[0], &h1, FPGA_OPEN_SHARED));
   EXPECT_EQ(FPGA_OK, xfpga_fpgaOpen(tokens_[0], &h2, FPGA_OPEN_SHARED));
   EXPECT_EQ(FPGA_OK, xfpga_fpgaClose(h1));
   EXPECT_EQ(FPGA_OK, xfpga_fpgaClose(h2));
 }
 
-INSTANTIATE_TEST_CASE_P(openclose_c, openclose_c_p, ::testing::ValuesIn(test_platform::keys(true)));
+INSTANTIATE_TEST_CASE_P(openclose_c, openclose_c_p, ::testing::ValuesIn(test_platform::platforms({"skx-p"})));
```

Comment:
Why are these tests exclusive to hw and only on mcp? They should be good for hw and mock for all platforms, right?

Reply:
Or I guess the only one missing is for dcp.